src/coreComponents/mesh/ObjectManagerBase.cpp

```diff
@@ -272,7 +272,7 @@ localIndex ObjectManagerBase::packImpl( buffer_unit_type * & buffer,
     std::set_difference( input.cbegin(), input.cend(), available.cbegin(), available.cend(), std::inserter( tmp1, tmp1.end() ) );
     if( !tmp1.empty() )
     {
-      GEOSX_LOG( "Wrappers \"" << stringutilities::join( tmp1, ", " ) << "\" were requested and are not available." );
+      GEOSX_LOG( "Wrappers \"" << stringutilities::join( tmp1, ", " ) << "\" were requested to " << getName() << " but are not available." );
```

Comment:
I think we could make it print only on rank 0. Unless it makes sense to know on which rank it failed.